### PR TITLE
Added new variants and tags. Fix #21. Fix #22. Fix #23. Fix #24. Prep…

### LINKED ISSUE #21: Introduce new meta-data columns
Disease mode of inheritance and Clinical repeat unit

### LINKED ISSUE #22: Ensure applicable HGNC-ids / symbols
Make sure Stranger and reference files use HGNC-ids that accurately reflect or at least can be tied to corresponding diseases (especially for filtering in Scout).

### LINKED ISSUE #23: Source
GeneReviews https://www.ncbi.nlm.nih.gov/books/NBK535148/ and others.
Two tags: pubmed id, gene review id.

### LINKED ISSUE #24: Other genes
Add HOXD13, HOXA13, FOXL2, LRP12, MUC1, EIF4A3, ... 
See https://www.ncbi.nlm.nih.gov/books/NBK535148/

### LINKED ISSUE #25: Add rarity statistic from normal population
It is straighforward to produce normal population calls for each locus. These should be employed to show the rarity of the present expansions size, e g as a quantile value for the less selected population.

### PR DESCRIPTION
**Pull request by dnil**
…arations to fix #25.

### CODE REVIEW

File: stranger/resources/variant_catalog_grch37.json [hunk 1]
```diff
@@ -47,32 +117,62 @@
...
     {
         "VariantType": "Repeat",
         "LocusId": "PHOX2B",
+        "HGNCId": 9143,
+        "InheritanceMode": "AD",
+        "DisplayRU": "GCN",
+        "SourceDisplay": "GeneReviews Internet 2019-11-07",
+        "Source": "GeneReviews",
+        "SourceId": "NBK535148",
+        "SweGenMean": 26,33033033,
```
Comment (moonso) [inline stranger/resources/variant_catalog_grch37.json:158]:
```suggestion
        "SweGenMean": 26.33033033,
```

File: stranger/resources/variant_catalog_grch37.json [hunk 2]
```diff
@@ -47,32 +117,62 @@
...
+        "SweGenStd": 11,87773598,
```
Comment (moonso) [inline stranger/resources/variant_catalog_grch37.json:159]:
```suggestion
        "SweGenStd": 11.87773598,
```

**Review by moonso — APPROVED** [review]
Wow! Nice one @dnil 🍇 
Small comment on decimals

**Comment by moonso** [comment]
So much activity here!! 🛩️

**Comment by dnil** [comment]
Ill add it to Scout next and see if I can’t get those filters up as well. 😅

**Pull request merged**